bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -64,7 +64,8 @@ class PyNeurodamus(PythonPackage):
     depends_on("py-h5py", type=("build", "run"))
     depends_on("py-numpy@1.24:", type=("build", "run"))
     depends_on("py-docopt", type=("build", "run"))
-    depends_on("py-libsonata", type=("build", "run"))
+    # TODO: Unpin libsonata for `neurodamus@3.6.0:`
+    depends_on("py-libsonata@:0.1.27", type=("build", "run"))
```

Comment:
Maybe something like this would be better?
depends_on("py-libsonata@:0.1.27", when="@:3.5.0", type=("build", "run"))
depends_on("py-libsonata", when="@3.6.0:", type=("build", "run"))

Reply:
That depends on the future and how you fix the issue. I can't know if `3.6.0` is compatible with `0.1.27`. Therefore, there's a TODO which gives a heads up to the dev releasing `3.6.0`.

More interesting is whether you want `develop` to be pinned too. Currently, it is.